### ISSUE TITLE
Clean up and vectorize new bot icons

### ISSUE THREAD

**Issue by lemonsaurus** [python-discord/branding#101]
I've created three rough bot icons for the new bots. I'm wondering if someone would like to polish them a little, turn them into vector, but stay within the same overall style. Simplification is allowed, if it makes sense to do so, and little pydisifications would also be fun to see.

Here are the icons as raster pngs.

## Sir Lancebot
This one is pretty rough - the lines are not very sharp and the tail is a complete disaster.
![image](https://user-images.githubusercontent.com/2098517/99529393-03314980-29a0-11eb-9f51-f33694537ee9.png)

## The Black Knight
This one I am quite happy with, probably just needs vectorization.
![image](https://user-images.githubusercontent.com/2098517/99529480-2a881680-29a0-11eb-849e-c455df6f0aea.png)

## Zoot
This one is pretty boring, and honestly doesn't have enough contrast to work well as an icon. The general idea of being a golden grail seems reasonable to me, but I'd be open to a complete reinterpretation or even a totally different icon with the same overall color profile (e.g. yellowish).
![image](https://user-images.githubusercontent.com/2098517/99529594-5c00e200-29a0-11eb-930c-bb5529a3e3cf.png)

**Comment by gustavwilliam** [comment]
I'm curious as to your reasoning behind a  yellow/orange logo for Zoot. To me, that signals money and some sort of economics, which isn't quite what I imagine for the bot. Is this purely meant to let all bots use different colors?

**Comment by lemonsaurus** [comment]
@gustavwilliam Grails are yellow. But I'm not married to yellow, I just want a color that's distinct from the other bots.

![image](https://user-images.githubusercontent.com/2098517/103457014-c923c900-4cfb-11eb-8a6b-7d51a069af2e.png)

Looking at the icons now, I think the Sir Lancebot and Black Knight icons are probably fine and that vectorization isn't crucial, but a new icon for Zoot seems more urgent. It has far too little contrast and isn't very nice.

I think if we can produce a better icon for Zoot, we can probably just close this issue.

**Comment by gustavwilliam** [comment]
That seems like a good plan. After seeing the bots on the server for a while now, I agree with your observations.

As for the Zoot logo, I think lowering the value of the yellow could go a long way for increasing the contrast. The overall shape could also be refined to more clearly resemble a grail, making it easier to understand at a smaller scale. From the last two months with the icon, it has looked more like this shape `)(` in a circle, than a grail to me.

In short, I think that the following would go a long way:
- Refine grail shape
- Decrease value of yellow

**Comment by lemonsaurus** [comment]
Sounds reasonable to me.
